pkg/cmd/install.go

```diff
@@ -235,6 +237,12 @@ func (o *installCmdOptions) install(_ *cobra.Command, _ []string) error {
 			platform.Spec.Build.HTTPProxySecret = o.httpProxySecret
 		}
 
+		if o.cache {
+			platform.Spec.Build.Cache = true
```

Comment:
Do we need the if statement? Can we not simplify to `platform.Spec.Build.Cache = o.cache`?

Reply:
thanks for the review, I have updated it as per your suggestions, I was trying to be defensive if someone passes non boolean values.